### DISCUSSION TITLE
weppy 1.3.5

### DISCUSSION THREAD

**Discussion by josejachuf** [emmett-framework/emmett#380] [General]
Hi @gi0baro 
About this [1] you are going to launch version 1.3.5 in pypi? 

jose

[1] https://github.com/emmett-framework/emmett/commit/14dd40036669d2b462f283074387bf1d28d81dce

**Comment by gi0baro** [comment]
@josejachuf Yes, I planned this for next week since I have to do it manually.

**Reply by josejachuf** [reply]
Hello @gi0baro, please do not forget this. Thanks

**Reply by gi0baro** [reply]
whoa! I'm sorry, I completely forgot this. Gonna do it shortly.
Out of curiosity: what prevents you from upgrading to Emmett?

**Reply by josejachuf** [reply]
> Out of curiosity: what prevents you from upgrading to Emmett?

I would like it, it is a medium application and you have to touch a lot and it is also installed in many places. But at some point I'm going to do it. 

Emmett seems great to me. Many people have not used something else, but when you try to do something with Emmett they tell you no, we use Django or in any case Flask. 

They do not stop at least try it. O The other answer is usually: "It is a small project, if you need something to whom you are going to consult"

It is true, it is smaller than flask. As for support I always got the answers and very fast

**Reply by gi0baro** [reply]
@josejachuf 1.3.5 is now available in pypi

**Reply by josejachuf** [reply]
Thanks!!
